content/collections/browser_sdk/en/browser-unified-sdk.md

```diff
@@ -134,7 +135,8 @@ All options from `@amplitude/analytics-browser` are supported. See the [Analytic
 
 ### Session Replay options
 
-All options from `@amplitude/plugin-session-replay-browser` are supported. See the [Session Replay documentation](/docs/session-replay/session-replay-standalone-sdk#configuration) for details.
+All options from `@amplitude/plugin-session-replay-browser` are supported. See the [Session Replay documentation](/docs/session-replay/session-replay-plugin#configuration) for details. Set `config.sessionReplay.sampleRate` to a non-zero value to enable session replay. 
```

Comment:
```suggestion
The Unified Browser SDK supports all options from `@amplitude/plugin-session-replay-browser`. See the [Session Replay Plugin documentation](/docs/session-replay/session-replay-plugin#configuration) for more information. Set `config.sessionReplay.sampleRate` to a non-zero value to enable session replay. 

Sample Rate controls the rate at which Amplitude captures session replays. For example, if you set `config.sessionReplay.sampleRate` to `0.5`, Session Replay captures roughly half of all sessions.
```

Reply:
Thanks Mark for reviewing it. I agree we should be more clear here. But let's keep the example code short to help customers set up quickly.